_includes/common/sql/statement-bundle-warning.md

```diff
@@ -1,2 +1,4 @@
-{{site.data.alerts.callout_danger}} Statement bundles can contain unredacted user data including histograms and placeholders that contain real data samples, statements that have real data, and database schema which could be sensitive. Be careful when generating and sharing statement bundles.
+{{site.data.alerts.callout_danger}}Statement bundles can contain unredacted user data including histograms and placeholders that contain real data samples, statements that have real data which can contain PII information, and database schema that could be sensitive. Be careful when generating and sharing statement bundles.
+
+<br><br>To allow or disallow a role from seeing other roles' <a href="{{ link_prefix }}ui-statements-page.html#diagnostics">statements diagnostics bundles</a>, set the <code>NOVIEWACTIVITYREDACTED</code> <a href="{{ link_prefix }}create-role.html#role-options">role option</a>.
```

Comment:
This role is not about seeing other roles bundles or not, is about seeing bundles in general, independent if it was created by another role. 
Also, the NO___ option is usually more to remove the role option, so a user should grant or revoke the VIEWACTIVITYREDACTED role.

The sentence should be "To allow or disallow a role from seeing statements diagnostics bundle, set the VIEWACIVITYREDACTED role option"